lua/starfall/libs_cl/material.lua

```diff
@@ -346,11 +346,12 @@ end
 --- Gets a texture from a material
 -- @param path The path of the material (don't include .vmt in the path)
 -- @param texture The texture key to get
--- @return The texture's name
+-- @return The texture's name or nil if texture key isn't found
```

Comment:
Returns nil if the material isn't found, error texture if the texture isn't found?

Reply:
If what's passed to `GetTexture`, so the `texture` parameter of this function isn't found on the given material, then it returns nil.